examples/resources/env0_team_environment_assignment/resource.tf

```diff
@@ -0,0 +1,31 @@
+data "env0_template" "template" {
+  name = "Template Name"
+}
+
+data "env0_project" "project" {
+  name = "Default Organization Project"
+}
+
+resource "env0_environment" "environment" {
+  name        = "environment"
+  project_id  = data.env0_project.project.id
+  template_id = data.env0_template.template.id
+}
+
+resource "env0_team" "team" {
+  name = "team"
+}
+
+resource "env0_custom_role" "custom_role" {
+  name = "custom-role-sample"
+  permissions = [
+    "VIEW_PROJECT",
```

Comment:
change permissions to be ENVIRONMENT RELATED.. like view-env, edit-env-settings

Reply:
Hi @yarivg - could you please explain this comment?